src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -236,8 +415,88 @@ describe('sidebar/store/modules/annotations', function() {
       );
     });
 
+    it('should set basic default properties on a new/empty annotation', () => {
+      store.dispatch(actions.createAnnotation({ id: 'myID' }, now));
+
+      const createdAnnotation = selectors.findAnnotationByID(
+        store.getState(),
+        'myID'
+      );
+
+      assert.equal(createdAnnotation.created, now.toISOString());
```

Comment:
This assumes that the Date does not advance in-between the `beforeEach` call and the `createAnnotations` call. That is not a safe assumption unless you are using fake clocks. You could instead check that created date is within some threshold of the `now` value or between the timestamp at the `beforeEach` call and some timestamp after the `createAnnotations` call. Or use fake clocks to make sure the "current date" doesn't advance.